[2019-08-29 19:12:39,040] WARN Client session timed out, have not heard from server in 42320ms for sessionid 0x1000f91b0420013 (org.apache.zookeeper.ClientCnxn)
[2019-08-29 19:12:45,073] INFO Client session timed out, have not heard from server in 42320ms for sessionid 0x1000f91b0420013, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
[2019-08-29 19:12:39,045] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:12:45,075] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:12:39,041] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:12:45,080] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:12:46,979] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-08-29 19:12:47,419] INFO Socket connection established to localhost/127.0.0.1:2181, initiating session (org.apache.zookeeper.ClientCnxn)
[2019-08-29 19:12:47,082] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error connecting to node 192.168.1.13:9093 (id: 1 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:476)
	at java.base/sun.nio.ch.Net.connect(Net.java:468)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:280)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:258)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:945)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:288)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:12:47,081] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error connecting to node 192.168.1.13:9092 (id: 0 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:476)
	at java.base/sun.nio.ch.Net.connect(Net.java:468)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:280)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:258)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:945)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:288)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:12:47,431] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:12:47,430] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:12:47,425] INFO [ZooKeeperClient Kafka server] Session expired. (kafka.zookeeper.ZooKeeperClient)
[2019-08-29 19:12:47,425] WARN Unable to reconnect to ZooKeeper service, session 0x1000f91b0420013 has expired (org.apache.zookeeper.ClientCnxn)
[2019-08-29 19:12:47,467] INFO Unable to reconnect to ZooKeeper service, session 0x1000f91b0420013 has expired, closing socket connection (org.apache.zookeeper.ClientCnxn)
[2019-08-29 19:12:47,425] INFO EventThread shut down for session: 0x1000f91b0420013 (org.apache.zookeeper.ClientCnxn)
[2019-08-29 19:12:47,469] INFO [ZooKeeperClient Kafka server] Initializing a new session to localhost:2181. (kafka.zookeeper.ZooKeeperClient)
[2019-08-29 19:12:47,469] INFO Initiating client connection, connectString=localhost:2181 sessionTimeout=6000 watcher=kafka.zookeeper.ZooKeeperClient$ZooKeeperClientWatcher$@7b4c50bc (org.apache.zookeeper.ZooKeeper)
[2019-08-29 19:12:47,467] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:12:47,431] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:12:47,641] INFO Creating /brokers/ids/2 (is it secure? false) (kafka.zk.KafkaZkClient)
[2019-08-29 19:12:47,641] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-08-29 19:12:48,186] INFO Socket connection established to localhost/0:0:0:0:0:0:0:1:2181, initiating session (org.apache.zookeeper.ClientCnxn)
[2019-08-29 19:12:48,406] INFO Session establishment complete on server localhost/0:0:0:0:0:0:0:1:2181, sessionid = 0x1000f91b0420018, negotiated timeout = 6000 (org.apache.zookeeper.ClientCnxn)
[2019-08-29 19:12:48,902] INFO Stat of the created znode at /brokers/ids/2 is: 13951,13951,1567086168426,1567086168426,1,0,0,72074712439717912,194,0,13951
 (kafka.zk.KafkaZkClient)
[2019-08-29 19:12:49,239] INFO Registered broker 2 at path /brokers/ids/2 with addresses: ArrayBuffer(EndPoint(192.168.1.13,9094,ListenerName(PLAINTEXT),PLAINTEXT)), czxid (broker epoch): 13951 (kafka.zk.KafkaZkClient)
[2019-08-29 19:13:19,999] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:13:20,071] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:13:20,196] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:13:20,197] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:13:23,366] INFO [GroupMetadataManager brokerId=2] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2019-08-29 19:13:52,077] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:13:52,078] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:13:52,204] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:13:52,204] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:14:24,088] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:14:24,088] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:14:24,211] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:14:24,212] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:14:37,261] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:14:37,261] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:14:37,261] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:14:46,301] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:14:46,301] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:14:46,301] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:14:54,430] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:14:54,430] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:14:54,430] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:14:56,217] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:14:56,217] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:15:07,640] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:15:07,640] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:15:07,641] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:15:13,703] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:15:13,703] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:15:13,703] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:15:19,966] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:15:19,966] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:15:19,966] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:15:28,225] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:15:28,225] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:15:35,041] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Connection to node 0 (/192.168.1.13:9092) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:15:35,041] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:15:35,041] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:15:41,691] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Connection to node 0 (/192.168.1.13:9092) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:15:41,691] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:15:41,691] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:15:50,912] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Connection to node 0 (/192.168.1.13:9092) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:15:50,912] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:15:50,913] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:15:51,977] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:15:51,977] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:15:56,991] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Connection to node 0 (/192.168.1.13:9092) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:15:56,991] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:15:56,991] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:16:03,081] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Connection to node 0 (/192.168.1.13:9092) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:16:03,120] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:16:03,120] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:16:13,124] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:16:13,124] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:16:13,124] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:16:16,241] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Connection to node 0 (/192.168.1.13:9092) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:16:16,242] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:16:16,242] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:16:22,957] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Connection to node 0 (/192.168.1.13:9092) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:16:22,957] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:16:22,957] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:16:22,957] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:16:22,957] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:16:22,957] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:16:29,022] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:16:29,022] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:16:29,022] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:16:29,022] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Connection to node 0 (/192.168.1.13:9092) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:16:29,023] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:16:29,023] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:16:36,003] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:16:36,003] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:16:36,003] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:16:36,003] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Connection to node 0 (/192.168.1.13:9092) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:16:36,004] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:16:36,004] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:17:08,012] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:17:08,012] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:17:08,012] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:17:08,012] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:17:14,192] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Connection to node 0 (/192.168.1.13:9092) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:17:14,192] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:17:14,192] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:17:14,192] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:17:14,193] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:17:14,193] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:17:20,722] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Connection to node 0 (/192.168.1.13:9092) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:17:20,722] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:17:20,722] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:17:27,746] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:17:27,746] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:17:27,746] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:17:52,732] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:17:52,776] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:17:59,751] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:17:59,751] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:18:01,843] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Connection to node 0 (/192.168.1.13:9092) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:18:01,843] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:18:01,844] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:18:31,760] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:18:31,760] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:18:33,853] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:18:33,853] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:19:03,771] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:19:03,771] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:19:05,860] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:19:05,860] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:19:11,085] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:19:11,085] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:19:11,085] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:19:20,124] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Connection to node 0 (/192.168.1.13:9092) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:19:20,124] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:19:20,125] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:19:26,149] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Connection to node 0 (/192.168.1.13:9092) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:19:26,149] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:19:26,149] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:19:32,176] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:19:32,176] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:19:32,176] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:19:35,275] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Connection to node 0 (/192.168.1.13:9092) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:19:35,275] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:19:35,275] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:19:42,112] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Connection to node 0 (/192.168.1.13:9092) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:19:42,112] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:19:42,112] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:19:42,112] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:19:42,112] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:19:42,112] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:19:48,135] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Connection to node 0 (/192.168.1.13:9092) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:19:48,136] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:19:48,136] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:19:54,225] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Connection to node 0 (/192.168.1.13:9092) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:19:54,225] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:19:54,225] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:20:07,376] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Connection to node 0 (/192.168.1.13:9092) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:20:07,376] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:20:07,376] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9092 (id: 0 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:20:14,125] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:20:14,125] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:20:35,495] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:20:35,542] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:20:35,542] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:20:39,388] INFO [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error sending fetch request (sessionId=1982657551, epoch=INITIAL) to node 0: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:20:39,388] WARN [ReplicaFetcher replicaId=2, leaderId=0, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic128-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic123-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61]), topic132-1=(fetchOffset=15, logStartOffset=15, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic131-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[52]), topic129-0=(fetchOffset=117, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[56]), topic126-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[61])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1982657551, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-29 19:20:42,680] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-29 19:20:42,753] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037343974, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-29 19:20:42,754] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=258, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[78]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[78]), test-kafka-broker-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[81]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[92]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[73]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[97]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[103]), topic130-1=(fetchOffset=369, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[71]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[97]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[75]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[79])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037343974, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
